### DISCUSSION TITLE
Mapping for the action "Accept next word of inline suggestion"

### DISCUSSION THREAD

**Discussion by MatejBransky** [hrsh7th/nvim-cmp#1579] [Ideas]
Hi, I would like to get a similar action as is in the VS Code where you can accept only part of the suggestion which is especially great for the Github Copilot if you don't agree with the whole suggestion but with few modifications you can basically co-op with the Copilot.

<img width="1325" alt="image" src="https://github.com/hrsh7th/nvim-cmp/assets/20944757/3c0ce83c-292f-42ba-bc83-fb3bd5559653">

https://github.com/hrsh7th/nvim-cmp/assets/20944757/55dc0656-7b83-4663-882f-c9f029b2ff09

I don't know if that's possible right now because I don't know how would I achieve the same with the current mapping. Is there any solution for it or does it require some changes in the cmp itself?

**Comment by Shougo** [comment]
Please see this.

https://github.com/hrsh7th/nvim-cmp/issues/1536

**Reply by MatejBransky** [reply]
Thanks and sorry for the duplicate 🙏 . Have a nice day! 🙂  I'm closing this as a duplicate of the #1536 .
